src/main/java/by/yahor_kulesh/dao/TicketDAO.java

```diff
@@ -13,12 +13,20 @@
 import java.sql.Types;
 import java.util.UUID;
 
+@Component
 public class TicketDAO{
+
+    private final ConnectionConfig connectionConfig;
+
     private ResultSet rs = null;
 
+    public TicketDAO(ConnectionConfig connectionConfig) {
+        this.connectionConfig = connectionConfig;
+    }
+
     public void insert(TicketEntity ticket){
         try(
-                Connection con = ConnectionConfig.getConnection();
+                Connection con = connectionConfig.getConnection();
                 PreparedStatement stat = con.prepareStatement("INSERT INTO ticket(id,user_id,ticket_type,creation_date) VALUES (?,?,?,?)")
```

Comment:
you can use:
```java
PreparedStatement statement = dataSource.getConnection().prepareStatement(sqlCommand)
```
Method `getConnection()` will be already configured by Spring under the hood for specific datasource

Reply:
done